packages/vehicle-lifecycle-network/lib/manufacturer.js

```diff
@@ -12,109 +12,95 @@
  * limitations under the License.
  */
 
+/* global emit */
```

Comment:
I was expecting to see the other global functions like getAssetRegistry here as in the other examples